docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -81,13 +81,17 @@ This cause model is designed to simulate the basic structure of the risk factor
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-1. In vivarium, 'uncomplicated DM Type 1 and Type 2' is 'Moderate', which is different from how GBD 2017 is modelling it. In the future, severity splits will be revisited using disability weights. 
+- In vivarium, 'uncomplicated DM Type 1 and Type 2' is 'Moderate', which is different from how GBD 2017 is modelling it. In the future, severity splits will be revisited using disability weights. 
 
-3. EMR is greater than zero for severe diabetes only.
+- This model has durations for moderate and severe DM that are too long because they arrive to that disease state immediately. Simulants in this model will not progress from moderate to severe.
 
-4. Case definition cross-walks on FPG and HbA1c: GBD 2017 assumed that HbA1c >6.5% was equivalent to FPG >126 mg/dL.
+- This model assumes that EMR from 'moderate' and 'severe' disease states are equal.
 
-5. There are no incidence or transitions between states in this model. In the future, transition between states will be revisited using incidence of diabetes mellitus and remission.
+- This model assumes that Remission from 'moderate' to 'susceptible' and 'severe' to 'susceptible' are the same value.
+
+- Case definition cross-walks on FPG and HbA1c: GBD 2017 assumed that HbA1c >6.5% was equivalent to FPG >126 mg/dL. 
+
+- There are no incidence or transitions between states in this model. In the future, transition between states will be revisited using incidence of diabetes mellitus and remission.
```

Comment:
There is incidence into both "moderate" and "severe" (prevalence weighted incidence (parent cause incidence)) and remission (equal for both "moderate" and "severe").  There is no state-to-state transition for the reasons cited above.